tests/Shared/Aspire.Workload.Testing.targets

```diff
@@ -39,30 +39,20 @@
 
       <NuGetsToBuildForWorkloadTesting
                       Include="$(_ShippingPackagesDir)Aspire.Dashboard.Sdk.$(NETCoreSdkRuntimeIdentifier).nupkg"
-                      Project="$(RepoRoot)eng/dashboardpack/dashboardpack.csproj"
+                      Project="$(RepoRoot)eng/dashboardpack/Aspire.Dashboard.Sdk.$(NETCoreSdkRuntimeIdentifier).csproj"
                       Properties="@(DefaultPropertiesForNuGetBuild, ';');@(_PropertiesForNuGetBuilds, ';')"
                       Descriptor="Dashboard pack"/>
 
       <NuGetsToBuildForWorkloadTesting
                       Include="$(_ShippingPackagesDir)Aspire.Hosting.Orchestration.$(NETCoreSdkRuntimeIdentifier).nupkg"
-                      Project="$(RepoRoot)eng/dcppack/dcppack.csproj"
+                      Project="$(RepoRoot)eng/dcppack/Aspire.Hosting.Orchestration.$(NETCoreSdkRuntimeIdentifier).csproj"
                       Properties="@(DefaultPropertiesForNuGetBuild, ';');@(_PropertiesForNuGetBuilds, ';')"
                       Descriptor="Dcp pack"/>
     </ItemGroup>
 
     <PropertyGroup>
       <_CombinedProps>@(DefaultPropertiesForNuGetBuild, ';');@(_PropertiesForNuGetBuilds, ';')</_CombinedProps>
     </PropertyGroup>
-    <!-- FIXME:
-      Explicitly invoke Build on these two projects so the GenerateTextReplacementFiles target gets triggered.
-      This can be removed once the workloadtesting targets support a custom list of targets to invoke
-    -->
-    <MSBuild Projects="$(RepoRoot)eng/dcppack/dcppack.csproj"
-              Properties="$(_CombinedProps)"
-              Targets="Restore;Build" />
-    <MSBuild Projects="$(RepoRoot)eng/dashboardpack/dashboardpack.csproj"
-              Properties="$(_CombinedProps)"
-              Targets="Restore;Build" />
```

Comment:
Removing this breaks local command line builds:
1. `./build.sh -pack`
2. `dotnet build tests/workloads.proj`
3. `dotnet test tests/Aspire.Workload.Tests/Aspire.Workload.Tests.csproj`

`/Users/ankj/dev/a2/artifacts/bin/Aspire.Workload.Tests/Release/net8.0/nuget-cache-Net80/aspire.dashboard.sdk.osx-arm64/8.2.0-dev/buildTransitive/Aspire.Dashboard.Sdk.osx-arm64.props(2,3): error MSB4019: The imported project "/Users/ankj/dev/a2/artifacts/bin/Aspire.Workload.Tests/Release/net8.0/nuget-cache-Net80/aspire.dashboard.sdk.osx-arm64/8.2.0-dev/build/@PackageName@.props" was not found. Confirm that the expression in the Import declaration "../build/@PackageName@.props" is correct, and that the file exists on disk. [/private/var/folders/dj/z2lsb9l9087356sgyzrlklvr0000gn/T/testroot/aspire-aspire-xunit/aspire-aspire-xunit.AppHost/aspire-aspire-xunit.AppHost.csproj]`